packages/botonic-plugin-ner/README.md

```diff
@@ -0,0 +1,75 @@
+# Botonic Plugin NER
+
+## What does this plugin do?
+
+Botonic Plugin NER uses the trained models defined by the user to recognize named entities inside the input text.
+
+After detecting the text locale, and using the corresponding model to recognize the entities, extra information about the recognized entities is added to the input message.
```

Comment:
First, the text locale is detected. Then, by using the relevant model, extra information about the recognized entities is added to the input message.